masschallenge/django-accelerator
# AC-5978: Add help text in Additional Industries in Startup Profile and Expert Profile page.

**Changes introduced in [AC-5978](https://masschallenge.atlassian.net/browse/AC-5978)**:
- add help text to 'Additional industries' fields

**How to test**:
- Ensure both accelerate and impact are running
- checkout AC-5978 in django-accelerator and run `make migrations`
- visit `http://localhost:8181/startup/add/` and `http://localhost:8181/accounts/register/expert/` and note the additional help text under the `Additional industries` fields

<img width="377" alt="screen shot 2018-11-12 at 18 12 40" src="https://user-images.githubusercontent.com/9318050/48356401-3b15a000-e6a7-11e8-9624-20be15688066.png">
